gollum-rjgit_adapter.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |s|
   s.summary     = %q{Adapter for Gollum to use RJGit at the backend.}
   s.description = %q{Adapter for Gollum to use RJGit at the backend.}
 
-  s.add_runtime_dependency "rjgit", "> 5.7.0", "~> 5.7.0"
+  s.add_runtime_dependency "rjgit", "~> 6.1"
```

Comment:
Required for `ConfigConstants::CONFIG_INIT_SECTION`.

Reply:
RJGit 6.2 was just released.

Reply:
Great! Still 6.1 is the minimum required, so think I'll just keep it like this?

Reply:
Sure. 👍